ui/lib/core/addon/components/search-select.hbs

```diff
@@ -68,7 +68,7 @@
               {{get selected this.idKey}}
             </small>
           {{else}}
-            <div>
+            <div class="list-item-text">
```

Comment:
Let's tyuse helpers instead of defining a new nested class.

If the intent is to have the following styling:
```
    max-width: 200px;
    overflow-x: hidden;
    text-overflow: ellipsis;
    margin-right: 5px;
```

you should be able to chain pre-existing helpers together and maybe make a new helper where needed. 

So something like 
`<class='text-overflow-ellipsis has-right-margin-xxs one-fourth-width ...`
This is not complete, but you can kind of gather the idea. Feel free to make new helpers before you make new component classes. The helper-classes directory has some well named files that give you a sense of what files cover what kind of helper classes.

Hit me up if you have any questions!